16/umbraco-cms/fundamentals/backoffice/property-editors/built-in-umbraco-property-editors/rich-text-editor/blocks.md

```diff
@@ -0,0 +1,212 @@
+# Blocks
+
+Blocks enable editors to insert structured content elements directly into the Rich Text Editor (RTE). Blocks are [Element Types](../../../../../../data/defining-content/#element-types) and can be configured with custom properties, styling, and behavior.
+
+Blocks can be added to the Rich Text Editor when:
+
+*   Available Blocks are specified as part of the Rich Text Editor Data Type configuration
```

Comment:
Extra spaces between bullet an text is not necessary 🙏 

Could you please go through the PR an remove these extra spaces?

```suggestion
* Available Blocks are specified as part of the Rich Text Editor Data Type configuration
```